README.md

```diff
@@ -54,6 +58,7 @@ Sorted alphabetically
 | [:link:](https://www.catminingexperience.com) | Caterpillar - MINExpo 2016 | `1.6` | | |
 | [:link:](https://www.cerberusapp.com) | Cerberus | `1.8` | | |
 | [:link:](https://www.cloudstitch.com) | Cloudstitch | `1.8` | | |
+| [:link:](https://github.com/F4IF/ctree-demo) | Collaboration Tree (Demo) | `1.7` | :heavy_check_mark: | [Source](https://github.com/F4IF/ctree-demo)  |
```

Comment:
I double and triple checked, but I don't think there is any demo. The user has to clone the repo and serve locally AFAIK.